templates/rapidsai/Base.dockerfile.j2

```diff
@@ -15,7 +15,7 @@ FROM ${FROM_IMAGE}:${RAPIDS_VER}-cuda${CUDA_VER}-base-${LINUX_VER}-py${PYTHON_VE
 
 ARG DASK_SQL_VER
 
-RUN gpuci_mamba_retry install -y -n rapids -c dask/label/dev \
+RUN gpuci_mamba_retry install -y -n rapids -c conda-forge \
```

Comment:
minor change required here. `conda-forge` is included in our `.condarc` file in the parent images, so we can remove this `-c` flag entirely now.

Reply:
Done